_source/logzio_collections/_log-sources/lambda-extensions.md

```diff
@@ -0,0 +1,213 @@
+
+    a. Repeat step 2 to add another layer.
+  
+    b. Select the `Specify an ARN` option, then select the ARN that's compatible with the extension version you chose from the dependencies table in the last tab of this topc, and paste it in the textbox. 
+  
+    c. Click `Add`.
+
+##### Configure the extension parameters
+
+Add the environment variables to the function, according to the environment variables table in the last tab of this topic.
+
+##### Run the function
+
+Run the function. It may take more than one run of the function for the logs to start shipping to your Logz.io account.
+
+</div>
+
+#### Deleting the extension
+
+- To delete the **extension layer**: In your function page, go to the **layers** panel. Click `edit`, select the extension layer, and click `save`.
+- To delete the **extension dependencies layer**: In your function page, go to the **layers** panel, click `edit`, select the extension dependencies layer, and click `save`.
+- To delete the extension's **environment variables**: In your function page, select the `Configuration` tab, select `Environment variables`,  click `edit`, and remove the variables that you added for the extension.
+
+
+</div>
+<!-- tab:end -->
+
+<!-- tab4:start -->
+<div id="tables">
+
+### Environment Variables
+
+| Name | Description |Required/Default|
+| --- | --- | --- |
+| `LOGZIO_LOGS_TOKEN` | Your Logz.io logs shipping [token](https://app.logz.io/#/dashboard/settings/general). | Required |
+| `LOGZIO_REGION` |  Two-letter region code, or blank for US. This determines your listener URL. You can find your region code in the [Regions and URLs table](https://docs.logz.io/user-guide/accounts/account-region.html#regions-and-urls). | Default: (left blank for US)|
+| `LOGS_EXT_LOG_LEVEL` |  Log level of the extension. Can be set to one of the following: `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`. |Default: `INFO` |
+| `ENABLE_EXTENSION_LOGS` |  Set to `true` if you wish the extension logs will be shipped to your Logz.io account. | Default: `false` |
+| `ENABLE_PLATFORM_LOGS` | The platform log captures runtime or execution environment errors. Set to `true` if you wish the platform logs will be shipped to your Logz.io account. | Default: `false` |
+| `THREAD_TIMEOUT` | Execution timeout for the threads that parse and send the logs. | Default: `5` (seconds)|
+| `LOGZIO_CUSTOM_LISTENER` | Use if you have a custom Logz.io listener endpoint). Will override `LOGZIO_REGION`. | Optional |
+
+
+### Lambda extension versions
+
+| Version | Supported Runtimes | AWS ARN |
+| --- | --- | --- |
+| 0.0.1 | python 3.7, python 3.8 | `arn:aws:lambda:<<YOUR-AWS-REGION-CODE>>:486140753397:layer:LogzioLambdaExtensionLogs:1` |
+
+### ARN for extension dependencies
+
+|Compatible with extension versions | Imports | AWS ARN |
+| --- | --- | --- |
+| 0.0.1 | `requests` | `arn:aws:lambda:<<YOUR-AWS-REGION-CODE>>:486140753397:layer:LogzioLambdaExtensionLogsLibs:1` |
```

Comment:
@YajB The AWS ARN column - since the values are long, it splits to a multiline. Is there a way to make it look better?

Reply:
@mirii1994 
I would love to!  Haven't yet found the right syntax. 
@Doron-Bargo , are you aware of how to do this,  while complying with the syntax requirements for the pages within the product?

Reply:
Line breaks still happen, even w/o table formatting.